### PR TITLE
Updating README

### PR DESCRIPTION
**Pull request by jtaleric**
## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] Optimization
- [x] Documentation Update

## Description
Adding metric-endpoints.yaml example

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,18 @@ For example:
...
+- endpoint: prometheus-k8s-openshift-monitoring.apps.rook.devshift.org 
+  profile: metrics-profiles/metrics.yml
+  alertProfile: alerts-profiles/alerts.yml
+- endpoint: thanos-querier-openshift-monitoring.apps.rook.devshift.org 
+  token: {{ my_token }} 
+  profile: metrics-profiles/thanos-metrics.yml
+```
+
+`my_token` can be captured by running `oc whoami -t`
+
```
Comment (rsevilla87) [inline README.md:92]: When using system:admin the comand `oc whoami -t` doesn't work, a valid token can be fetched from the prometheus SA.

```suggestion
  token: {{ .TOKEN }} 
  profile: metrics-profiles/thanos-metrics.yml
```

`my_token` can be captured by running `TOKEN=$(oc sa new-token -n openshift-monitoring prometheus-k8s)`

```

File: README.md [hunk 2]
```diff
@@ -78,6 +78,18 @@ For example:
 kube-burner-ocp cluster-density-v2 --iterations=1 --churn-duration=2m0s --churn-cycles=2 --es-index kube-burner --es-server https://www.esurl.com:443 --metrics-endpoint metrics-endpoints.yaml
 ```
 
+metrics-endpoints.yaml
+```
+- endpoint: prometheus-k8s-openshift-monitoring.apps.rook.devshift.org 
+  profile: metrics-profiles/metrics.yml
+  alertProfile: alerts-profiles/alerts.yml
+- endpoint: thanos-querier-openshift-monitoring.apps.rook.devshift.org 
+  token: {{ my_token }}
```
Comment (rsevilla87) [inline README.md:87]:
```suggestion
  token: {{ .TOKEN }} 
```

Comment (jtaleric) [reply README.md:87]: I wasn't trying to template it, but sure!

**Review by rsevilla87 — APPROVED** [review]
lgtm!

**Pull request merged**